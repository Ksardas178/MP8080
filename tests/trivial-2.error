line 3: <WARNING> too less arguments. Expected 3
