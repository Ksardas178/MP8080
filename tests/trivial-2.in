MOV 1 2
MVI 1 2
LXI 1 2 3
LDA 1 2 //2-byte address
LDAX 1
STA 1 2 //2-byte address
STAX 1
IN 1
OUT 1
XCHG
XTHL
LHLD 1 2