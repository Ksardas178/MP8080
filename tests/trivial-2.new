line 6: <ERROR> wrong register name recieved
line 7: <ERROR> expected 2 argument(s)
line 8: <ERROR> wrong register name recieved
line 10: <ERROR> wrong register name recieved
line 11: <ERROR> wrong register name recieved
line 12: <ERROR> wrong register name recieved
line 12: <ERROR> expected 2 argument(s)

[30;1mCode analysis results:
[0m______________________________
00000000: MOV 1 2
00000001: MVI 1 2
00000010: LXI 1 2
00000011: LDA 1 2
00000100: LDAX 1 1
00000101: STAX 1 1
00000110: OUT 1 1
