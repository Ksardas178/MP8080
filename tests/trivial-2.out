
[33;1mWarnings: 1
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: MOV 1 2
00000001: MVI 1 2
00000010: LXI 0 1 2
00000011: LDA 1 2
00000100: LDAX 1
00000101: STA 1 2
00000110: STAX 1
00000111: IN 1
00001000: OUT 1
00001001: XCHG
00001010: XTHL
00001011: LHLD 1 2
